### ISSUE TITLE
Erroneous path in winpython 32bit 2.7.12.3Zero

### ISSUE THREAD

**Issue by theilen** [winpython/winpython#392]
Winpython-32bit-2.7.12.3Zero seems to create wrong paths in Windows 7 64bit.

When starting "Jupyter Notebook.exe", the run fails because Windows cannot find the provided path. The one it searches is indeed strange:

"C:\WinPython-32bit-2.7.12.3Zero\scripts\..\python-2.7.12\scripts\jupyter-notebook.exe"

Even without the unusual "..", there is no jupyter-notebook.exe in python-2.7.12\scripts.

The problem seems to persist regardless of the path winpython is extracted to. And it seems to be present in the 64bit version as well.

**Comment by stonebig** [comment]
Hi @theilen 

there is no jupyter package by default in Winpython-32bit-2.7.12.3**Zero**, only this:

`````
## WinPython 2.7.12.3Zero 

The following packages are included in WinPython v2.7.12.3Zero .

### Tools

Name | Version | Description
-----|---------|------------


### Python packages

Name | Version | Description
-----|---------|------------
[Python](http://www.python.org/) | 2.7.12 | Python programming language with standard library
[docopt](http://pypi.python.org/pypi/docopt) | 0.6.2 | Pythonic argument parser, that will make you smile
[idlex](http://pypi.python.org/pypi/idlex) | 1.13 | IDLE Extensions for Python
[jedi](http://pypi.python.org/pypi/jedi) | 0.9.0 | An autocompletion tool for Python that can be used for text editors
[pip](http://pypi.python.org/pypi/pip) | 8.1.2 | A tool for installing and managing Python packages
[prompt-toolkit](http://pypi.python.org/pypi/prompt-toolkit) | 1.0.7 | Library for building powerful interactive command lines in Python
[ptpython](http://pypi.python.org/pypi/ptpython) | 0.35 | Python REPL build on top of prompt_toolkit
[pygments](http://pygments.org) | 2.1.3 | Generic syntax highlighter for general use in all kinds of software
[pythonnet](http://pypi.python.org/pypi/pythonnet) | 2.1.0 | .Net and Mono integration for Python
[setuptools](http://pypi.python.org/pypi/setuptools) | 25.1.6 | Download, build, install, upgrade, and uninstall Python packages - easily
[six](http://pypi.python.org/pypi/six) | 1.10.0 | Python 2 and 3 compatibility utilities
[sqlite-bro](http://pypi.python.org/pypi/sqlite-bro) | 0.8.11 | a graphic SQLite Client in 1 Python file
[wcwidth](http://pypi.python.org/pypi/wcwidth) | 0.1.7 | Measures number of Terminal column cells of wide-character codes
[winpython](http://winpython.github.io/) | 1.6.20160816 | WinPython distribution tools, including WPPM (package manager)
````

**Comment by theilen** [comment]
So this is what the zero stands for.
You should think about making that clear somewhere prior to the download page, and you should maybe also think of losing the executables in the main folder then.
Thanks for the clarification!
